spec/lib/asset_sources_spec.rb

```diff
@@ -114,34 +120,37 @@
       let(:manifest_content) { nil }
 
       it 'returns an empty array' do
-        expect(AssetSources.get_assets('missing')).to eq([])
+        expect(asset_sources.get_assets('missing')).to eq([])
       end
     end
 
     it 'loads the manifest once' do
-      expect(AssetSources).to receive(:load_manifest).once.and_call_original
+      expect(asset_sources).to_not receive(:load_manifest)
 
-      AssetSources.get_assets('application')
-      AssetSources.get_assets('input')
+      asset_sources.get_assets('application')
+      asset_sources.get_assets('input')
     end
 
     context 'uncached manifest' do
-      before do
-        allow(AssetSources).to receive(:cache_manifest).and_return(false)
+      let(:asset_sources) do
+        AssetSources.new(
+          manifest_path: manifest_file.path, cache_manifest: false,
+          i18n_locales: [:en, :es, :fr]
+        )
       end
 
       it 'loads the manifest' do
-        expect(AssetSources).to receive(:load_manifest).twice.and_call_original
+        expect(asset_sources).to receive(:load_manifest).twice.and_call_original
 
-        AssetSources.get_assets('application')
-        AssetSources.get_assets('input')
+        asset_sources.get_assets('application')
+        asset_sources.get_assets('input')
       end
     end
   end
 
   describe '.get_integrity' do
```

Comment:
tiny nit, this is now an instance method
```suggestion
  describe '#get_integrity' do
```